app.js

```diff
@@ -273,12 +322,11 @@ function renderMarkdown(content, returnElement = false) {
     const rendered = md.render(content);
     tempDiv.innerHTML = rendered;
 
-    const footnotes = tempDiv.querySelector('.footnotes');
```

Comment:
why did u remove this part? i didnt ask u to remove it. what i want is in the reference section there's the footnotes part and then followed by the favicons of visitedURLs